src/recon_buildblock/ProjMatrixByBinUsingRayTracing.cxx

```diff
@@ -646,44 +654,51 @@ calculate_proj_matrix_elems_for_one_bin(
   */
   const float z_position_of_first_LOR_wrt_centre_of_TOR =
     (-sampling_distance_of_adjacent_LORs_z/(2*num_lors_per_axial_pos)*
-      (num_lors_per_axial_pos-1))
-    - origin.z();
-  float offset_in_z = 
-    z_position_of_first_LOR_wrt_centre_of_TOR
-    +(max_index.z()+min_index.z())/2.F * voxel_size.z();
+     (num_lors_per_axial_pos-1));
+  float offset_in_z = z_position_of_first_LOR_wrt_centre_of_TOR;
 
   if (tantheta==0)
     {
       // make sure we don't ray-trace exactly between 2 planes
       // z-coordinate (in voxel units) will be
-      //  (t_in_mm+offset_in_z)/voxel_size.z();
+      //  (m_in_mm+offset_in_z)/voxel_size.z();
       // if so, we ray trace first to the voxels at smaller z, but will add the 
       // other plane later (in add_adjacent_z)
-      if (fabs(modulo((t_in_mm+offset_in_z)/voxel_size.z(),1.F)-.5)<.001)
+      if (fabs(modulo((m_in_mm+offset_in_z)/voxel_size.z(),1.F)-.5)<.001)
         offset_in_z -= .1F*voxel_size.z();
     }
 
 
   // use FOV which is slightly 'inside' the image to avoid
   // index out of range
 #ifdef STIR_PMRT_LARGER_FOV
-  const float fovrad_in_mm = 
-    min((min(max_index.x(), -min_index.x())+.45F)*voxel_size.x(),
-        (min(max_index.y(), -min_index.y())+.45F)*voxel_size.y()); 
+  CartesianCoordinate3D<float> max_pos =
+    density_info_sptr
+      ->get_relative_coordinates_for_indices(max_index + 0.45F);
```

Comment:
I think best to use `physical`. At some point, we will allow x/y-offset shifts. The "relative" coordinates won't see this, and therefore we would use the wrong FOV

Reply:
agree, I'd like to phase out all relative coordinates, either removing the methods or changing them to private.

Reply:
let's just make them private (or protected) in a first (2nd?) stage